packages/coreutils/src/poll.ts

```diff
@@ -549,6 +562,62 @@ export namespace Poll {
   export const MAX_INTERVAL = 2147483647;
 }
 
+/**
+ * A poll that only schedules ticks manually.
```

Comment:
What if I call the other functions on `Debouncer` like `start()`, `stop()`, or `refresh()`? It seems like that would be a no-op, since the interval is `infinity`, but having those functions still available seems weird.

I am not sure I really understand the utility of having `Debouncer` be a base class of `Poll`. That brings in a ton of functionality and API surface area that is then more-or-less ignored. It also seems to have some confusing semantics.

How does one interpret this
```typescript
debouncer = new Debouncer(factory, 1000);
console.log(debouncer.frequency.interval); // "Infinity"
console.log(debouncer.interval); // 1000
```

Reply:
Yeah, I considered calling it `debounce` and `invoke()` ... or maybe I need to bite the bullet and call it `debounceInterval`.

Or just dispense with this class as overkill. It really comes down to whether we want to `stop()`. `refresh()` would actually be an immediate poll tick scheduled.

At the least, it seems this class should be private, but possibly it is superfluous.

Reply:
I would hate to make you use camelCase.